openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.succeeded.json

```diff
@@ -0,0 +1,39 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Completed",
+    "actor": {
+        "id": "http://localhost:18000/u/honor",
+        "name": "honor",
+        "type": "Person"
+    },
+    "eventTime": "2018-10-17T13:08:41.590Z",
+    "extensions": {
+        "extra_fields": {
+            "accept_language": "en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
+            "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+            "event_source": "server",
+            "event_type": "edx.course.enrollment.upgrade.succeeded",
+            "host": "tasawernawaz.sandbox.edx.org",
+            "ip": "116.58.62.58",
+            "page": null,
+            "path": "/shoppingcart/postpay_callback/",
+            "session": "f3029807d5ac0b71b59b53e9e789f0fc"
+        }
+    },
+    "id": "urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada",
+    "object": {
+        "id": "https://tasawernawaz.sandbox.edx.org/shoppingcart/payment_fake/",
+        "type": "Entity",
```

Comment:
This is a very generic type. Is there no specific Entity type for this event?

Reply:
i have changed the event, kindly give me another review